dim-testsuite/Makefile

```diff
@@ -14,23 +16,37 @@ all: install db test
 	mkdir ${OUTDIR}
 	mkdir ${LDIR}
 
-install:
+install-deps:
 	${PYTHON} -m venv ${VDIR}
 	${VPIP} install -r ../dim/requirements.txt
+	${VPIP} install -r ../ndcli/requirements.txt
+
+install-deps-local: install-deps
+	${VPIP} install ../dim
+	${VPIP} install ../dimclient
+	${VPIP} install ../ndcli
+
+install-deps-local-editable: install-deps
 	${VPIP} install -e ../dim
 	${VPIP} install -e ../dimclient
-	${VPIP} install -r ../ndcli/requirements.txt
 	${VPIP} install -e ../ndcli
 
-db:
+install: install-deps-local
+
+install-dev: install-deps-local-editable
+	${VPIP} install -r ../dim/requirements-dev.txt
+	${VPIP} install -r ../ndcli/requirements-dev.txt
+
+db-clear:
+	${VDIR}/bin/manage_db clear
+
+db: db-clear
 	mysql -h127.0.0.1 -P3307 -updns -ppdns pdns1 < ./testenv/pdns.sql
 	mysql -h127.0.0.1 -P3307 -updns -ppdns pdns2 < ./testenv/pdns.sql
-	${VDIR}/bin/manage_db clear
 
-test:
+test: db-clear
```

Comment:
The database cleaning is done by runtest.py before every testcase file. See [runtest.py#329](https://github.com/1and1/dim/blob/master/dim-testsuite/runtest.py#L329).

Reply:
Right you are, was not aware.

I've removed that dependency for `pytest` as well, as that does clear the DB as well, see https://github.com/1and1/dim/blob/da7c2b2b1a0926abacb31d74121d0433fddea7b7/dim-testsuite/tests/util.py#L23-L28